### PR TITLE
[SYCLomatic] Support migration of cub::{LoadDirectBlocked, LoadDirectStriped}

### PR DESCRIPTION
**Pull request by yihwang-nv**

### CODE REVIEW

File: clang/lib/DPCT/APINames_CUB.inc [hunk 1]
```diff
@@ -190,11 +190,11 @@ ENTRY_MEMBER_FUNCTION(cub::DeviceSegmentedSort, cub::DeviceSegmentedSort, Stable
 // Thread and thread block I/O
 ENTRY(cub::ThreadLoad, cub::ThreadLoad, true, NO_FLAG, P4, "Successful")
 ENTRY(cub::ThreadStore, cub::ThreadStore, true, NO_FLAG, P4, "Successful")
-ENTRY(cub::LoadDirectBlocked, cub::LoadDirectBlocked, false, NO_FLAG, P4, "Comment")
+ENTRY(cub::LoadDirectBlocked, cub::LoadDirectBlocked, true, NO_FLAG, P4, "Successful")
 ENTRY(cub::LoadDirectBlockedVectorized, cub::LoadDirectBlockedVectorized, false, NO_FLAG, P4, "Comment")
 ENTRY(cub::StoreDirectBlocked, cub::StoreDirectBlocked, false, NO_FLAG, P4, "Comment")
 ENTRY(cub::StoreDirectBlockedVectorized, cub::StoreDirectBlockedVectorized, false, NO_FLAG, P4, "Comment")
-ENTRY(cub::LoadDirectStriped, cub::LoadDirectStriped, false, NO_FLAG, P4, "Comment")
+ENTRY(cub::LoadDirectStriped, cub::LoadDirectStriped, true, NO_FLAG, P4, "Comment")
```
Comment (tangjj11) [inline clang/lib/DPCT/APINames_CUB.inc]:
```suggestion
ENTRY(cub::LoadDirectStriped, cub::LoadDirectStriped, true, NO_FLAG, P4, "Successful")
```

**Review by tangjj11 — CHANGES_REQUESTED** [review]

File: clang/lib/DPCT/APINames_CUB.inc (hunk 1 above)
Comment (yihwang-nv) [reply clang/lib/DPCT/APINames_CUB.inc]: done

**Review by tangjj11 — APPROVED** [review]

**Review by abhilash1910 — COMMENTED** [review]
LGTM! Thanks

**Review by ziranzha — CHANGES_REQUESTED** [review]
Should keep backward compatibility.

**Review by ziranzha — APPROVED** [review]

**Pull request merged**